### ISSUE TITLE
Error running example

### ISSUE THREAD

**Issue by DavidHribek** [jonkhler/s2cnn#60]
Hello,

I am trying to run mnist example, but i am getting many errors running script /examples/mnist/run.py. Seems like i have bad version of some packages but cant figure out which versions are needed. Fixing one error leads to another error..

Some errors:
`x = torch.fft(torch.stack((x, torch.zeros_like(x)), dim=-1), 2)
TypeError: 'module' object is not callable`

`RuntimeError: view_as_complex is only supported for half, float and double tensors, but got a tensor of scalar type: ComplexFloat`

I have installed all the dependencies:

```
conda create --name cuda9 python=3.6 
conda activate cuda9

# s2cnn deps
#conda install pytorch torchvision cuda90 -c pytorch # get correct command line at http://pytorch.org/
conda install -c anaconda cupy  
pip install pynvrtc joblib

# lie_learn deps
conda install -c anaconda cython  
conda install -c anaconda requests  

# shrec17 example dep
conda install -c anaconda scipy  
conda install -c conda-forge rtree shapely  
conda install -c conda-forge pyembree  
pip install "trimesh[easy]"  
```

`python setup.py install`

Any help? Thanks.

**Comment by Fenyutanchan** [comment]
Yes, because the `torch.fft` is a module which is not callable, and in the comment above this line, it says that this may be `torch.fft.rfft`. But this correction doesn't work too, because of the new problem below this line. The solution to this problem is using the cuda, with which these codes are not reachable.

**Comment by DavidHribek** [comment]
Thank you. After changing a few lines, the code works for gpu.

**Comment by DavidHribek** [comment]
@mariogeiger I have created the PR. It is just one line that fixed the error.
